packages/monaco/src/typings/monaco/index.d.ts

```diff
@@ -364,6 +364,25 @@ declare module monaco.actions {
      */
     // https://github.com/theia-ide/vscode/blob/standalone/0.19.x/src/vs/platform/actions/common/actions.ts#L142
     export const MenuRegistry: IMenuRegistry;
+
+    // https://github.com/theia-ide/vscode/blob/standalone/0.19.x/src/vs/platform/actions/common/actions.ts#L112
+    export interface IMenuActionOptions {
+        arg?: any;
+        shouldForwardArgs?: boolean;
+    }
+
+    // https://github.com/theia-ide/vscode/blob/standalone/0.19.x/src/vs/platform/actions/common/actions.ts#L246
+    export class MenuItemAction {
+        readonly item: ICommandAction;
```

Comment:
Do we need to copy all properties?  I don't see them to be used.

Reply:
I removed properties.
I use this class only to check `actions[0] instanceof monaco.actions.MenuItemAction`

Reply:
probably constructor is not necessary as well?

Reply:
so, just empty class?
:-)